package.json

```diff
@@ -18,8 +18,10 @@
     "fs": "0.0.1-security",
     "handlebars": "^4.0.6",
     "jira-miner": "^0.3.3",
+    "kujira-data-miner": "^0.3.0-alpha.4",
```

Comment:
needs to be updated to current version (alpha.7), but this may already be changed in related PR's that are open 

edit: just checked the other open PRs, and they all call for alpha.4 which is old/ partially broken - alpha.7 has resolved all those issues & the version to use